extras/parse_travis_log.py

```diff
@@ -367,7 +367,7 @@ def makebuild_summary(log_filename, msg_make_section_start,
     # with some warnings, this would be a good point to re-set the
     # expected_warnings variable conditionally for that build_type.
 
-    nest_warning_re = re.compile(f'{build_dir}.*: warning:')
+    nest_warning_re = re.compile(f'.* ({build_dir}.*: warning:.*)')
     known_warnings = [
```

Comment:
Wondering if this should be a set instead of a list. For a 4-element collection it probably does not make a difference and list may even be faster since it is simpler—and we don't want this list to become long in any case. So maybe leave as is.

Reply:
Thanks for sharing your thoughts. I'll leave it as it is then ;-)